rgdata.c

```diff
@@ -1568,7 +1568,7 @@ void data_window_print(data_window_t *dat)
         printf("   \"gridwidth\":%d, \"gridheight\":%d,\n", dat->gridwidth, dat->gridheight);
     if (dat->type == wintype_Graphics)
         printf("   \"graphwidth\":%d, \"graphheight\":%d,\n", dat->gridwidth, dat->gridheight);
-    printf("   \"left\":%d, \"top\":%d, \"width\":%d, \"height\":%d }",
+    printf("   \"left\":%.1f, \"top\":%.1f, \"width\":%.1f, \"height\":%.1f }",
```

Comment:
Other places were using one decimal place so that's what I used, but probably more would be beneficial? Can it use as many decimal places as were calculated?